### ISSUE TITLE
Dark mode: Text content has insufficient contrast against its background

### ISSUE THREAD

**Issue by iamrafan** [microsoft/accessibility-insights-windows#843]
**Describe the bug**
"Automated checks" text in default state (when no app is scanned) and the fix message when no failures are detected, have insufficient contrast against their backgrounds.

**To Reproduce**
1. Make sure color setting for default app mode in windows is set to 'Dark'
2. Launch AI-Windows
3. Navigate to Tests section in the left menu bar without scanning any app
4. Observe the text "Automated checks" under fast pass has insufficient contrast against the background
5. Navigate back to live inspect and run tests on a section of an app that might not have any issues
6. Go to "Results in UI Automation Tree"
7. Observe that the message under "Fix the following" in the right pane has insufficient contrast against the background

**Expected behavior**
Text content to have contrast ratio of at least 4.5:1 against its background

**Actual behavior**
The contrast ratio for text falls short of 4.5:1

**Screenshots or .GIF**
![4 1](https://user-images.githubusercontent.com/22068434/87987123-f30d6a00-ca92-11ea-822d-23f349fee580.png)
![4 2](https://user-images.githubusercontent.com/22068434/87987143-fc96d200-ca92-11ea-8638-d78e2590607b.png)

**Desktop**
 - OS: Windows 10 Enterprise, Version 1903
 - Accessibility Insights for Windows Version: Production Release: v 1.1.1248.1

**Comment by DaveTryon** [comment]
@iamrafan, this is fixed in release [1.1.1380.01](https://github.com/microsoft/accessibility-insights-windows/releases/tag/v1.1.1380.01). Please regress and close at your convenience.

**Comment by iamrafan** [comment]
Validated the fix in Canary Release: v1.1.1380.01
